index.test.js

```diff
@@ -390,7 +405,30 @@ p {
   :root {
     --text: black;
   }
-}`, 
+}`,
   MockProps
   )
 })
+
+it('Can jit a light and dark with custom adaptive prop', async () => {
```

Comment:
```suggestion
it('Can jit a light and dark color with a custom adaptive prop parameter', async () => {
```

is this suggestion the same as what you meant?